### PR TITLE
update quickstart to add k3d

### LINKED ISSUE #1292: Quickstart Prerequisite Suggestions
https://code.habd.as/mirrors/postgres-operator/src/branch/master/docs/quickstart.md#prerequisites

Currently KIND and Minikube are suggested. Given the operator runs multi-node I propose adding K3s as well. Like KIND and unlike Minikube K3s supports multi-node local development. Adding K3s here should result in improved compatibility with edge deployments longer-term.

### PR DESCRIPTION
**Pull request by balibebas**
add info on k3s 🙏🏼 

closes #1292

### CODE REVIEW

**Comment by balibebas** [comment]
Or would it be better to list k3d as a more light-weight docker alternative to KiND?

**Comment by FxKu** [comment]
Hm , yeah the description is a bit vague. I have no experience with `k3s`. How can I (easily) test it locally? Would be good if the short description addresses this.

**Comment by balibebas** [comment]
> Hm , yeah the description is a bit vague. I have no experience with `k3s`. How can I (easily) test it locally? Would be good if the short description addresses this.

I switched the description to k3d and added more detail. To test locally install k3d, start docker, run `k3d cluster create` and then install the operator.

File: docs/quickstart.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ solutions:
   single-node K8s cluster inside a VM (requires KVM or VirtualBox),
 * [kind](https://kind.sigs.k8s.io/), which allows creating multi-nodes K8s
   clusters running on Docker (requires Docker)
+* [k3d](https://k3d.io), which allows creating multi-node K8s clusters running
+  under Docker (requires Docker)
```
Comment (FxKu) [inline docs/quickstart.md:17]:
```suggestion
* [kind](https://kind.sigs.k8s.io/) or [k3d](https://k3d.io), which allow creating multi-node K8s
  clusters running on Docker (requires Docker)
```

Comment (FxKu) [reply docs/quickstart.md:17]: lets combine it when it does the same

Comment (balibebas) [reply docs/quickstart.md]: done

**Comment by balibebas** [comment]
will squash when ready

**Comment by FxKu** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Comment by balibebas** [comment]
squashed. ready for merge

**Comment by FxKu** [comment]
wasn't necessary. We can squash on merge. Now we have to approve again, but could have told you :smiley:

**Comment by FxKu** [comment]
:+1:

**Comment by sdudoladov** [comment]
:+1:

**Pull request merged**